src/lib/containers/UserCardListRotate.tsx

```diff
@@ -137,6 +137,11 @@ const UserCardListRotate: React.FunctionComponent<UserCardListRotateProps> = ({
   return (
     <div className="UserCardListRotate bootstrap-4-backport">
       {isLoading && loadingScreen}
+      {!isLoading && userIds.length === 0 && (
+        <p className="font-italic">
+          No one was found.
```

Comment:
Needed to show something here, since the section is always shown when I use this component on the Details page.  When shown on a Home page, we should never encounter this case.